pkgs/applications/editors/ne/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, ncurses, texinfo, tetex, perl, ghostscript }:
+
+
+stdenv.mkDerivation {
+  name = "ne-3.0.1";
+  version = "3.0.1";
+  src = fetchurl {
+    url = https://github.com/vigna/ne/archive/3.0.1.tar.gz;
```

Comment:
`fetchFromGitHub` will fetch this for you and saves you from a gotcha that comes from using the standard fetcher with github URLs: GitHub will cache the generated archives for a while, but then will regenerate them when they fall out of the cache. When it regenerates the archive, it will have a different hash, even though the contents are identical. `fetchFromGitHub` saves you from such mundane concerns!

```nix
fetchFromGitHub {
  owner = "vigna";
  repo = "ne";
  rev = "3.0.1";
  # use a dummy, see what it tells you instead, and then replace with the real one
  sha256 = "0gq1j9b0d0rykql3jzdb2yn4adj0rrcsvqrmj3dz001ir57ilsgc";
}
```